contrib/charts/dragonfly/templates/_pod.tpl

```diff
@@ -91,7 +91,13 @@ containers:
     {{- include "dragonfly.volumemounts" . | trim | nindent 4 }}
     {{- if .Values.passwordFromSecret.enable }}
     env:
+    {{- $appVersion := .Chart.AppVersion }}
+    {{- $imageTag := .Values.image.tag | default $appVersion }}
+    {{- if semverCompare ">=14.0.0" $imageTag }}
```

Comment:
You mean `<=14.0.0` here right? Cause we deprecated for `DFLY_requirepass`. With the code above, if version is greater than 14.0.0 it will use `DFLY_PASSWORD`

Reply:
Good catch. Thanks. Will update.